mungegithub/mungers/submit-queue.go

```diff
@@ -1305,7 +1324,11 @@ func (sq *SubmitQueue) servePriorityInfo(res http.ResponseWriter, req *http.Requ
       <li>PR with no release milestone will be considered after any PR with a milestone</li>
     </ul>
   </li>
-  <li>PR number</li>
+  <li>First time at which the LGTM label was applied.
```

Comment:
That's effectively how it works for everyone today, since it goes off of
number... If people abuse this in that way we can switch to something else.

On Thu, Aug 18, 2016 at 3:31 PM, krousey <notifications@github.com> wrote:

> In mungegithub/mungers/submit-queue.go
> <https://github.com/kubernetes/contrib/pull/1563#discussion_r75401421>:
>
> > @@ -1305,7 +1324,11 @@ func (sq *SubmitQueue) servePriorityInfo(res http.ResponseWriter, req *http.Requ
> >        <li>PR with no release milestone will be considered after any PR with a milestone</li>
> >      </ul>
> >    </li>
> > -  <li>PR number</li>
> > +  <li>First time at which the LGTM label was applied.
>
> So I should just create my PRs with LGTM and DO_NOT_MERGE and then remove
> them, right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1563/files/aee73d1142f4fa0657f7e5338b23f4e4ee5f2b3c#r75401421>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAnglmbIbMkgB1QYX_1c89r9bTz6a-cjks5qhN1LgaJpZM4Jn-Mc>
> .
>